docs/src/pages/guides/notifications.md

```diff
@@ -173,4 +171,24 @@ body: {
 }
 ```
 
+## Slack Incoming Webhook
+
+Monika supports Slack Incoming Webhook. To enable notification via Slack, you must have a `Slack's Incoming Webhook URL`, which you can see in their own documentation to get that. Please see [Sending messages using Incoming Webhooks](https://api.slack.com/messaging/webhooks).
```

Comment:
You can remove this part: `which you can see in their own documentation to get that`. 
Then, `Please consult [Sending messages using Incoming Webhooks](https://api.slack.com/messaging/webhooks) documentation.`